src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,15 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request. This property contains the
+        /// escaped path and full query, as well as other request targets such as * for OPTIONS
```

Comment:
I was equally confused :smile: But "encoded" here means they are URL-encoded i.e. the `%`s will be there.